### PR TITLE
*: use newest master mio.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -55,6 +54,9 @@ git = "https://github.com/pingcap/kvproto"
 [dependencies.tipb]
 git = "https://github.com/pingcap/tipb.git"
 
+[dependencies.mio]
+git = "https://github.com/carllerche/mio.git"
```
Comment (ngaut) [inline Cargo.toml]: We should not depend on master

**Comment by ngaut** [comment]
Rest LGTM

File: Cargo.toml (hunk 1 above)
Comment (siddontang) [inline Cargo.toml]: only master fix kqueue bug.

**Comment by ngaut** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @BusyJay

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**